src/xcode/ENA/ENA/Source/Developer Menu/Features/DMDebugRiskCalculationViewController.swift

```diff
@@ -101,12 +86,14 @@ class DMDebugRiskCalculationViewController: UIViewController {
 		encoder.dateEncodingStrategy = .iso8601
 
 		textView.text += "\n\nConfiguration:\n\n"
-		if let data = try? encoder.encode(mostRecentRiskCalculationConfiguration), let mostRecentRiskCalculationConfiguration = String(data: data, encoding: .utf8) {
+		if let data = try? encoder.encode(mostRecentRiskCalculationConfiguration),
```

Comment:
We should consider proper error handling, even for dev code. This might bring up potential de-/serialization issues earlier.